modules/core/src/main/java/org/apache/ignite/internal/processors/cache/CacheMetricsImpl.java

```diff
@@ -803,6 +815,7 @@ public EntriesStatMetrics getEntriesStat() {
                         offHeapBackupEntriesCnt += cacheSize;
 
                     size = (int)offHeapEntriesCnt;
+                    sizeLong = offHeapEntriesCnt;
```

Comment:
Is it possible to move it beyond the for cycle?

Reply:
Done.